runtime/src/prioritization_fee.rs

```diff
@@ -136,13 +136,13 @@ pub enum PrioritizationFeeError {
 /// block; and the minimum fee for each writable account in all transactions in this block. The only relevant
 /// write account minimum fees are those greater than the block minimum transaction fee, because the minimum fee needed to land
 /// a transaction is determined by Max( min_transaction_fee, min_writable_account_fees(key), ...)
-#[derive(Debug)]
+#[derive(Debug, Default)]
 pub struct PrioritizationFee {
-    // The minimum prioritization fee of transactions that landed in this block.
-    min_transaction_fee: u64,
+    // Prioritization fee of transactions that landed in this block.
+    transaction_fees: Vec<u64>,
 
-    // The minimum prioritization fee of each writable account in transactions in this block.
-    min_writable_account_fees: HashMap<Pubkey, u64>,
+    // Prioritization fee of each writable account in transactions in this block.
+    writable_account_fees: HashMap<Pubkey, Vec<u64>>,
```

Comment:
Concern of memory footprint led to only record `min` and `prune_irrelevant_writable_accounts` when bank is OCed. Possible abusive scenario: 4000 txs/ block, 128 accounts/tx, then writable_account_fees would require *additional* sizeof(u64) * 4000 * 128 = 4MB per un-OCed block. I need more convincing that recording every transactions' and every accounts' prio fee will not be an attack vector.  

Could histogram, or rolling EMA with variance (https://en.wikipedia.org/wiki/Moving_average#Exponentially_weighted_moving_variance_and_standard_deviation) be alternative? Perhaps user can query "ema plus n stddev"?